CMakeLists.txt

```diff
@@ -128,6 +130,12 @@ if(TRITON_ENABLE_CC_HTTP OR TRITON_ENABLE_CC_GRPC OR TRITON_ENABLE_PERF_ANALYZER
   if(NOT ${TRITON_ENABLE_PERF_ANALYZER} AND ${TRITON_ENABLE_PERF_ANALYZER_C_API})
     message(FATAL_ERROR "Must set cmake argument 'TRITON_ENABLE_PERF_ANALYZER=ON' for using C API.")
   endif() # NOT TRITON_ENABLE_PERF_ANALYZER AND TRITON_ENABLE_PERF_ANALYZER_C_API
+  if(NOT ${TRITON_ENABLE_PERF_ANALYZER} AND ${TRITON_ENABLE_PERF_ANALYZER_TFS})
+    message(FATAL_ERROR "Must set cmake argument 'TRITON_ENABLE_PERF_ANALYZER=ON' for using TensorFlow Serving.")
```

Comment:
Change the error message as:
"TRITON_ENABLE_PERF_ANALYZER_TFS=ON requires TRITON_ENABLE_PERF_ANALYZER=ON"
Do the similar change in L131 and L137. Message should be direct otherwise it opens a bunch of questions.

Reply:
See here for example : https://github.com/triton-inference-server/server/blob/main/CMakeLists.txt#L93